File: Python/백준/Silver/2002. 추월/추월.py
import sys
input = sys.stdin.readline

n = int(input())
enter = [input().strip() for _ in range(n)]

count = 0
for _ in range(n):
    car = input().strip()
    if enter[0] != car:
        count += 1
    enter.remove(car)
print(count)

File: Java/백준/Silver/2002. 추월/추월.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(br.readLine());

        LinkedList<String> enter = new LinkedList<>();
        for (int i = 0; i < n; i++) {
            enter.add(br.readLine());
        }

        int count = 0;
        for (int i = 0; i < n; i++) {
            String car = br.readLine();
            if (!enter.getFirst().equals(car)) {
                count++;
            }
            enter.remove(car);
        }
        System.out.println(count);
    }
}